pyproject.toml

```diff
@@ -76,7 +81,7 @@ nipy_diagnose = 'nipy.cli.diagnose:main'
 
 [tool.ruff]
 line-length = 88
-target-version = 'py39'
+target-version = 'py38'
```

Comment:
Isn't ruff able to catch `python-requires = ">=3.8"`? I'll investigate that.

Reply:
Yes, it is. I forgot.